tests/layer_tests/pytorch_tests/test_pow.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('test_input', [(np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([[1, 1], [2, 2]], dtype=np.float32),),
+                                        (np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([2, 3], dtype=np.float32),),
+                                        (np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([2], dtype=np.float32),),
+                                        (np.array([5, 6], dtype=np.float32),
+                                         np.array([[1, 2], [3, 4]], dtype=np.float32),),
+                                        (np.array([5], dtype=np.float32),
+                                         np.array([[1, 2], [3, 4]], dtype=np.float32),)])
+class TestPow(PytorchLayerTest):
+    """
+    Input test data contains five test cases - elementwise power, broadcast exponent, one exponent,
+    broadcast base, one base.
+    """
+
+    def _prepare_input(self):
+        return self.test_input
+
+    def create_model(self):
+        class aten_pow(torch.nn.Module):
+
+            def forward(self, input_data, exponent):
+                return torch.pow(input_data, exponent)
```

Comment:
Looks like this only line will change from test to test. Could we implement test in more compact manner?

Reply:
I mean all tests.

Reply:
Not really just one line. For more complicated ops you would need to modify whole model. E.g. `__init__` method

Reply:
In this case we will have two lines of code: one in __init__ and one in test.